README.md

```diff
@@ -8,11 +8,11 @@ This is the GitHub page for the Detect plugin for Azure DevOps.
 
 
 ## Where can I get the latest release? ##
-Detect for Azure DevOps is available on the [Azure Marketplace](https://marketplace.visualstudio.com/items?itemName=synopsys-detect.synopsys-detect).
+Detect for Azure DevOps is available on the [Azure Marketplace](https://marketplace.visualstudio.com/items?itemName=blackduck-detect.blackduck-detect).
```

Comment:
TODO: confirm link works when plugin is published to marketplace

Reply:
@shantyk Lets change this to blackduck.blackduck-detect (I think the publisher name before the period(.) comes from marketplace entry )